administrator/modules/mod_login/tmpl/default.php

```diff
@@ -53,7 +54,7 @@ class="form-control"
 					type="password"
 					class="form-control input-full"
 					required="required"
-					autocomplete="current-password"
+					autocomplete="current-password"	
```

Comment:
```suggestion
					autocomplete="current-password"
```
Please remove the trailing tabulator which breaks code style again.